pkg/sloop/server/internal/config/config_test.go

```diff
@@ -16,8 +19,9 @@ func Test_loadFromJSONFile_Success(t *testing.T) {
 	err = json.Unmarshal(configFile, &expectedconfig)
 	assert.Nil(t, err)
 
-	out_config := loadFromFile(configfilename)
-	assert.Equal(t, out_config, &expectedconfig)
+	var outConfig *SloopConfig
+	outConfig = loadFromFile(configfilename, outConfig)
+	assert.Equal(t, outConfig, &expectedconfig)
```

Comment:
assert package expects first the `expected` value to be passed, and then the `actual` value.
Please fix what is being passed as expected, and what is passed as actual.

Documentation: 
> Equal func(t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{}) bool